portfolio/src/main/webapp/index.html

```diff
@@ -42,20 +42,12 @@ <h3>Skills</h3>
       <h3>Projects and Hackathons</h3>
       <dl>
         <dt>Be Smart Hackathon</dt>
-        <dd>Collaborated with a team of 3 students to design an application for American Airlines to improve customer service for passengers with disabilities. 
-<<<<<<< HEAD
```

Comment:
Looks like you have some things from the merge conflict that wasn't properly cleaned up.